roles/etcd_server_certificates/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Install etcd
+  action: "{{ ansible_pkg_mgr }} name=etcd state=present"
+  when: not etcd_is_containerized | bool
+
```

Comment:
with us now requiring 2.1+, couldn't we remove ansible_pkg_mgr invocations and replace with the generic package module?